sdk/core/azure-core/azure/core/utils/_connection_string_parser.py

```diff
@@ -0,0 +1,47 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import Mapping
+
+
+def parse_connection_string(conn_str, case_sensitive_keys=False):
+    # type: (str, bool) -> Mapping[str, str]
+    """Parses the connection string into a dict of its component parts, with the option of preserving case
```

Comment:
For some reason the documentation for this doesn't seem to be rendering?

Reply:
🤔 looking into it!

Reply:
I copied the `_connection_string_parser.py` file to `azure/core` and imported `parse_connection_string` in `azure/core/__init__.py`. The change showed up in the docs, so I don't think `_connection_string_parser.py` has errors that would cause rendering failure.

I also copied the `azure/core/tracing` folder to a `azure/core/tracing_copy` folder and assumed that if `tracing` is working, then `tracing_copy` should also render. This didn't work either, so I believe this might not be a problem with this code. Thoughts, @annatisch ?